oh my lord lmao

edit:

```
/home/runner/work/Cortex-Command-Community-Project/Cortex-Command-Community-Project/github/actions/osxcross/osxcross/target/bin/../macports/pkgs/opt/local/include/minizip/ioapi.h:74:10: fatal error: ints.h: No such file or directory
```

what the FUCK

--- a/.github/actions/osxcross/macports.patch
+++ b/.github/actions/osxcross/macports.patch
@@ -24,3 +24,12 @@ index 3e87735..5c48e4e 100755
      "$CACHE/$file.rmd160" "$CACHE/$file" 1>/dev/null
  
    if [ $? -ne 0 ]; then
+@@ -360,7 +362,7 @@ getPkgUrl()
+                grep -o -E '"version":"[^"]+"' | \
+                cut -d'"' -f4)
+ 
+-  pkgs=$(getFileStdout "$MIRROR/$pkgname/?C=M;O=A" | \
++  pkgs=$(getFileStdout "$MIRROR/$pkgname/?C=M;O=D" | \
+          grep -o -E 'href="([^"#]+)"' | \
+          cut -d'"' -f2 | grep '.tbz2$' | uniq)
+ 

--- a/.github/workflows/meson.yml
+++ b/.github/workflows/meson.yml
@@ -178,7 +178,7 @@ jobs:
           echo "::group::Installing pkg-config"
           sudo apt install pkg-config
           echo "::group::Installing mac deps"
-          omp install SDL3 onetbb lz4 libpng minizip luajit flac
+          omp --x86_64 install SDL3 onetbb lz4 libpng minizip luajit flac
           echo "OSXCROSS_PKG_CONFIG_PATH=${{env.OSXCROSS_TARGET}}/macports/pkgs/opt/local/libexec/onetbb/lib/pkgconfig" >> $GITHUB_ENV
           echo "::endgroup::"
           echo "::group::Installing meson"

